votor/src/voting_utils.rs

```diff
@@ -233,8 +249,14 @@ pub(crate) fn insert_vote_and_create_bls_message(
     let bank = context.root_bank_cache.root_bank();
     let bls_message = match generate_vote_tx(&vote, &bank, context) {
         GenerateVoteTxResult::BLSMessage(bls_message) => bls_message,
+        e @ GenerateVoteTxResult::NoAuthorizedVoter(_, _)
+        | e @ GenerateVoteTxResult::NoVoteState(_)
+        | e @ GenerateVoteTxResult::VoteAccountNotFound(_)
+        | e @ GenerateVoteTxResult::Tx(_) => {
+            return Err(VoteError::InvalidConfig(Box::new(e)));
+        }
         e => {
-            return Err(VoteError::GenerationError(Box::new(e)));
+            return Err(VoteError::TransientError(Box::new(e)));
```

Comment:
so is essentially everything either an InvalidConfig or TransientError? so generate_vote_message_and_push_to_votes() will essentially never bubble up an error?